### PR TITLE
Raise error for unknown parameter_values

### PR DESCRIPTION
**Pull request by tommaull**
added else statement to inform user of an invalid parameter set

# Description

Added else statement to inform user of an invalid parameter set in parameter_values.py

In response to #3063 - Raise error for unknown parameter sets

Fixes # (3063)

## Type of change

Please add a line in the relevant section of [CHANGELOG.md](https://github.com/pybamm-team/PyBaMM/blob/develop/CHANGELOG.md) to document the change (include PR #) - note reverse order of PR #s. If necessary, also add to the list of breaking changes.

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (back-end change that speeds up the code)
- [x] Bug fix (non-breaking change which fixes an issue)

# Key checklist:

- [x] No style issues: `$ pre-commit run` (see [CONTRIBUTING.md](https://github.com/pybamm-team/PyBaMM/blob/develop/CONTRIBUTING.md#installing-and-using-pre-commit) for how to set this up to run automatically when committing locally, in just two lines of code)
- [x] All tests pass: `$ python run-tests.py --all`
- [x] The documentation builds: `$ python run-tests.py --doctest`

You can run unit and doctests together at once, using `$ python run-tests.py --quick`.

## Further checks:

- [ ] Code is commented, particularly in hard-to-understand areas
- [ ] Tests added that prove fix is effective or that feature works

### CODE REVIEW

**Comment by tommaull** [comment]
Ran unit tests, FAILED (failures=1, skipped=62)
FAIL: test_symbol_visualise (test_expression_tree.test_symbol.TestSymbol)

Traceback (most recent call last):
  File "C:\Users\tom.maull\Desktop\PybammDev\PyBaMM\tests\testcase.py", line 29, in wrapped
    return method(*args, **kwargs)
  File "C:\Users\tom.maull\Desktop\PybammDev\PyBaMM\tests\unit\test_expression_tree\test_symbol.py", line 393, in test_symbol_visualise
    self.assertTrue(os.path.exists("test_visualize.png"))
AssertionError: False is not true

**Comment by tommaull** [comment]
Doc test result:
(pybamm) C:\Users\tom.maull\Desktop\PybammDev\PyBaMM>python run-tests.py --doctest
Checking if docs can be built.
Traceback (most recent call last):
  File "C:\Users\tom.maull\Desktop\PybammDev\PyBaMM\run-tests.py", line 427, in <module>
    run_doc_tests()
  File "C:\Users\tom.maull\Desktop\PybammDev\PyBaMM\run-tests.py", line 68, in run_doc_tests
    p = subprocess.Popen(
  File "C:\Users\tom.maull\AppData\Local\mambaforge\envs\pybamm\lib\subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Users\tom.maull\AppData\Local\mambaforge\envs\pybamm\lib\subprocess.py", line 1420, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
FileNotFoundError: [WinError 2] The system cannot find the file specified

File: pybamm/parameters/parameter_values.py [hunk 1]
```diff
@@ -60,10 +60,12 @@ def __init__(self, values, chemistry=None):
             self.update(values, check_already_exists=False)
         else:
             # Check if values is a named parameter set
-            if isinstance(values, str) and values in pybamm.parameter_sets:
+            if isinstance(values, str) and values in pybamm.parameter_sets.keys():
                 values = pybamm.parameter_sets[values]
                 values.pop("chemistry", None)
                 self.update(values, check_already_exists=False)
+            else:
+                print("Invalid Parameter Set Value")
```
Comment (rtimms) [inline pybamm/parameters/parameter_values.py:68]: could you raise a `ValueError` here? E.g.
```python3
else:
    raise ValueError("helpful message")
```

**Review by rtimms — CHANGES_REQUESTED** [review]
Thanks for taking this on! After you've updated the code to raise an error can you also add a test in [this file](https://github.com/pybamm-team/PyBaMM/blob/1a012e75953d4cd83e6705b4e5c250291fbe75fd/tests/unit/test_parameters/test_parameter_values.py) to confirm the error is raised when it should be. See [here](https://github.com/pybamm-team/PyBaMM/blob/1a012e75953d4cd83e6705b4e5c250291fbe75fd/tests/unit/test_parameters/test_parameter_values.py#L39) for an example.

The errors you are seeing locally are unrelated.

**Comment by rtimms** [comment]
@all-contributors please add @tommaull for code and tests

**Comment by tommaull** [comment]
changelog updated - sorry about that!

**Pull request merged**